package.json

```diff
@@ -57,8 +57,11 @@
         "@rollup/plugin-commonjs": "22.0.0",
         "@rollup/plugin-node-resolve": "13.3.0",
         "@rollup/plugin-typescript": "8.3.2",
+        "@storybook/addon-actions": "^6.4.22",
+        "@storybook/addon-docs": "^6.4.22",
         "@storybook/addon-essentials": "6.4.22",
         "@storybook/addon-interactions": "6.4.22",
+        "@storybook/addon-links": "^6.4.22",
```

Comment:
Do we need these other storybook adding? I don't see them in use yet?

Reply:
Removed